apps/condo/pages/ticket/[id]/index.tsx

```diff
@@ -106,6 +109,18 @@ export const TicketFileList: React.FC<ITicketFileListProps> = ({ files }) => {
         a.click()
     }
 
+    const handleFileDownload: (file: UploadFile) => void = async (file: UploadFile) => {
+        if (/.*\.(svg|pdf|txt)$/gi.test(file.name)) {
```

Comment:
:) we use to move static regexps (that are not changed) to constants in order not to compile it on every function call

Reply:
its done

Reply:
But I did not find similar regexes in constants :)